framework/DBUS.md

```diff
@@ -52,7 +52,7 @@ see get_all_alerts()
 
 ***
 
-##### get_alert(s: local_id) -> ssiasa(ssssbb)
+##### get_alert(s: local_id) -> ssiasa(ssssbb)sss
```

Comment:
the array signature doesn't have the extra `i` for the priority
I think:
`##### get_alert(s: local_id) -> ssiasa(ssssbbi)sss`